tools/atip/atip/web/web.py

```diff
@@ -407,8 +411,13 @@ def click_element_coords(self, x, y, key, display=True):
     def fill_element_by_key(self, key, text, display=True):
         element = self.__get_element_by_key(key, display)
         if element:
-            element.send_keys(text)
-            return True
+            try:
+                element.send_keys(text)
+                return True
+            except WebDriverException :
```

Comment:
I think adding workaround method into exception does not make sense, as the program cannot catch the exception if there is something wrong in try section.